tools/ddrgen/src/scanners/dwarf/AixDwarfParser.cpp

```diff
@@ -705,18 +283,19 @@ parseDeclarationLine (const string data, Parsed_Declaration_Data *parsedData)
 static int
 parseDwarfDie(const string line)
 {
-	
 	int ret = DW_DLV_OK;
 	/* get the type of the data of the line being parsed */
 	Parsed_Declaration_Data *parsedDeclarationData = new Parsed_Declaration_Data();
```

Comment:
This should probably not be a pointer. As it is, you have used "new" to allocate this, but never freed it, so it leaks memory. I suggest using this instead:
Parsed_Declaration_Data parsedDeclarationData = {0};
Dwarf_Half tag = parseDeclarationLine(line, &parsedDeclarationData);